### DISCUSSION TITLE
GetAccountCharacterModelsAsync cannot get the characters for VRM 1.0

### DISCUSSION THREAD

**Discussion by kyousuke916** [pixiv/vroid-sdk-developers#80] [Q&A]
Today I suddenly discovered that using DefaultApi.GetAccountCharacterModelsAsync() this API,
I got back the character list, which only VRM x.0, no VRM 1.0 characters,
But until yesterday it was possible to get the VRM 1.0 character,
I would like to ask what is going on? I am using VRoid SDK version 0.1.4.

**Comment by ytomi-pixiv** [comment]
Thanks for the report.

We have just released VRoid SDK0.2.0.
This will be an update that will allow you to import VRM1.0 models. [https://github.com/pixiv/vroid-sdk-developers/discussions/79](https://github.com/pixiv/vroid-sdk-developers/discussions/79)

Along with this update, we have made a change to not pass VRM1.0 models via API to the 0.1.4 and earlier versions that could not import VRM1.0.
If you wish to use VRM1.0 models, could you consider updating to VRoid SDK 0.2.0? Or if you want to use VRM1.0 with 0.1.4 or earlier versions, we would be happy to hear more about the background.

**Reply by kyousuke916** [reply]
Thank you very much
